### ISSUE TITLE
Improve visibility of Sign-up and Sign-in buttons

### ISSUE THREAD

**Issue by vaishaldsouza** [Wilcolab/Anythink-Market-pba5pk87#5]
The current grey color of the sign-up and sign-in buttons makes them difficult to see. They should be updated to a high contrast color to improve accessibility
